docs/pages/reference/cli.mdx

```diff
@@ -311,6 +319,30 @@ $ tsh recordings ls --from-utc=2022-11-01 --to-utc=2022-11-3
 $ tsh recordings ls --last=6h0m0s
 ```
 
+<Admonition
+  type="note"
+  title="Recorded Sessions Availability"
+  scope={["oss", "enterprise"]} scopeOnly
+>
+
+Recorded sessions are linked from the audit events to session recordings files. Self-hosted high availability Teleport
+deployments with directory storage for audit logs and session recordings can result in unavailable playback. That occurs
```

Comment:
>Self-hosted high availability Teleport
deployments with directory storage for audit logs and session recordings can result in unavailable playback. 

Can we rephrase? This sentence is long and hard to parse. Not sure how to edit or I'd offer a suggestion.

Worth noting though that in my case the issue wasn't with an HA setup and separate storage locations. Instead I suspect the issue was with rotation and older sessions being archived/compressed.

Reply:
rephrased @alexfornuto